### PR TITLE
 Synchronize CI workflow across projects

### PR DESCRIPTION
**Pull request by mgates3**
Fix minor differences so a multi-way diff
```
meld testsweeper/.github/workflows blaspp/.github/workflows lapackpp/.github/workflows
```
shows only significant changes, e.g., blaspp has devices.

Also change bitbucket.org to github.com.

### CODE REVIEW

**Review by KadirAkbudak — APPROVED** [review]
Looks good

File: .github/workflows/configure.sh [hunk 1]
```diff
@@ -1,37 +1,42 @@
...
+section "======================================== Environment"
+env
+
+section "======================================== Setup build"
 export color=no
 rm -rf ${top}/install
 if [ "${maker}" = "make" ]; then
     make distclean
     make config CXXFLAGS="-Werror" prefix=${top}/install
 fi
```
Comment (cayrols) [inline .github/workflows/configure.sh:30]: I am unfamiliar with lapackpp build process but is it normal that the `cmake` case clone `blaspp` and not the `make` case?

File: .github/workflows/setup_env.sh [hunk 2]
```diff
@@ -1,51 +1,66 @@
...
-    module load cmake
-    which cmake
-    cmake --version
-    cd build
-fi
+#-------------------------------------------------------------------------------
+quiet source /etc/profile
 
+hostname && pwd
+export top=`pwd`
```
Comment (cayrols) [inline .github/workflows/setup_env.sh:31]: Use `$()`.

**Review by cayrols — APPROVED** [review]
Looks good to me (just single style remark).

File: .github/workflows/configure.sh (hunk 1 above)
Comment (G-Ragghianti) [reply .github/workflows/configure.sh:30]: Yes, the makefile will automatically build blaspp in the blaspp subdirectory if the location of blaspp isn't provided.

Comment (mgates3) [reply .github/workflows/configure.sh:30]: Yes, the LAPACK++ `configure.py` automatically downloads and builds BLAS++, while with CMake we require that BLAS++ is already installed. It's feasible for CMake to also download BLAS++, as it does download TestSweeper if needed, but we haven't done that.

**Pull request merged**